crates/test-programs/tests/wasm_tests/runtime.rs

```diff
@@ -65,17 +65,14 @@ pub fn instantiate(
         })
         .collect::<Result<Vec<_>, _>>()?;
 
-    let instance = Instance::new(&module, &imports).context(format!(
+    let instance = Instance::new_wasi_abi(&module, &imports).context(format!(
         "error while instantiating Wasm module '{}'",
         bin_name,
     ))?;
 
-    instance
-        .get_export("_start")
-        .context("expected a _start export")?
-        .into_func()
-        .context("expected export to be a func")?
-        .call(&[])?;
+    if instance.is_some() {
```

Comment:
At first glance this looks like it should be `if instance.is_none() ...` (it looks like error-checking) but now I see that an Instance shouldn't be returned for commands--an explanatory comment might help?

Reply:
Comment added.